### PR TITLE
WebSockets samples for Java

### PR DESCRIPTION
**Pull request by labadav**
Adding doc samples for WebSocket and session affinity support beta release.

### CODE REVIEW

File: appengine/websockets/pom.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  limitations under the License.
+-->
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <version>1.0-SNAPSHOT</version>
+  <groupId>com.example.flexible</groupId>
+  <artifactId>websocket</artifactId>
+  <packaging>war</packaging>
+
+  <parent>
```
Comment (lesv) [inline appengine/websockets/pom.xml]: This parent should be:
```
  <!--
    The parent pom defines common style checks and testing strategies for our samples.
    Removing or replacing it should not affect the execution of the samples in anyway.
  -->
  <parent>
    <groupId>com.google.cloud.samples</groupId>
    <artifactId>shared-configuration</artifactId>
    <version>1.0.8</version>
  </parent>
```

**Comment by lesv** [comment]
ping

**Comment by lesv** [comment]
Ping again

**Comment by labadav** [comment]
Closing this as this needs to be re-done and re-tested. The team will re-submit this later. Thanks!

**Pull request closed without merging**